### PR TITLE
add setup target to makefile, other minor simplifications

### PR DESCRIPTION
**Pull request by mrod502**

### CODE REVIEW

**Review by ttarsi — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -13,11 +13,14 @@ The current version is not considered stable.
 
 ## Table of Contents
 
-- [Install](#install)
-- [Usage](#usage)
-- [Contributing](#contributing)
-- [License](#license)
-- [Acknowledgements](#acknowledgements)
+- [ｎｅｐｔｕｎｅ](#ｎｅｐｔｕｎｅ)
```
Comment (mrod502) [inline README.md:16]: My markdown extension did this, I think it's nice.

Comment (kevinhalliday) [reply README.md:16]: i does look cool but we don't have any other top level headers to it feels like we don't need 😬 am i a party pooper

**Review by kevinhalliday — APPROVED** [review]
Looks good! I vote to remove the top level link from table of contents tho

**Comment by ttarsi** [comment]
> Looks good! I vote to remove the top level link from table of contents tho

ya i agree it looks funky bc it's the only one

**Pull request merged**